src/Mobs/AggressiveMonster.h

```diff
@@ -20,7 +20,7 @@ class cAggressiveMonster :
 	virtual void InStateChasing(std::chrono::milliseconds a_Dt) override;
 	
 	virtual void EventSeePlayer(cEntity *) override;
-	virtual void Attack(std::chrono::milliseconds a_Dt);
+	virtual bool Attack(std::chrono::milliseconds a_Dt);
```

Comment:
I thought about it but I wasn't sure how to explain it; essentially it returns true if the attack was "successful" (even if it did no damage).

I'll add a comment later today